src/Microsoft.ML.Data/Evaluators/Metrics/RegressionMetrics.cs

```diff
@@ -15,10 +18,10 @@ public sealed class RegressionMetrics
         /// The absolute loss is defined as
         /// L1 = (1/m) * sum( abs( yi - y&apos;i))
         /// where m is the number of instances in the test set.
-        /// y'i are the predicted labels for each instance.
+        /// y&apos;i are the predicted labels for each instance.
```

Comment:
>&apos; [](start = 13, length = 6)

Prime is actualy &prime; (ampersand prime semicolon)

Reply:
Seems to be giving an error.

---
In reply to: [260537199](https://github.com/dotnet/machinelearning/pull/2700#discussion_r260537199) [](ancestors = 260537199)

Reply:
[https://www.w3.org/wiki/Common_HTML_entities_used_for_typography](https://www.w3.org/wiki/Common_HTML_entities_used_for_typography)

Should be standard?

---
In reply to: [260546858](https://github.com/dotnet/machinelearning/pull/2700#discussion_r260546858) [](ancestors = 260546858,260537199)

Reply:
No idea.

---
In reply to: [260599032](https://github.com/dotnet/machinelearning/pull/2700#discussion_r260599032) [](ancestors = 260599032,260546858,260537199)